controller/network/network.go

```diff
@@ -246,28 +260,43 @@ func (network *Network) LinkChanged(l *Link) {
 func (network *Network) BindService(srcR *Router, token string, serviceId string, peerData map[uint32][]byte) error {
 	log := pfxlog.Logger()
 
+	// TODO: Consider how we might replace this with a create endpoint ctrl handler, similar to the mgmt one
+
 	// 1: Find Service
-	if svc, found := network.serviceController.get(serviceId); found {
-		svc.Egress = srcR.Id
-		svc.EndpointAddress = "hosted:" + token
-		svc.PeerData = peerData
+	if _, found := network.serviceController.get(serviceId); found {
+		endpoint := &Endpoint{
+			Id:        token,
+			Service:   serviceId,
+			Router:    srcR,
+			Binding:   "edge",
```

Comment:
Do we want this defaulting to `edge` in the fabric code?

Reply:
I was thinking about this and I think we can remove the Bind/Unbind concept from the fabric, since it's really an edge concept, and replace it with just add/remove endpoint. Then the edge is responsible for all the semantics, including binding and address format. 

TBD is how routers can inject their keys for end to end encryption. Either way, though, bind isn't an appropriate mechanism for that.

Reply:
That sounds like a good avenue to explore. Adding/removing an endpoint to reflect SDK-hosted service presence makes a _lot_ more sense to me.